model/import/Report/ErrorReportFormatter.php

```diff
@@ -34,10 +34,7 @@ class ErrorReportFormatter extends AbstractReportFormatter
     public function format(array $report): Report
     {
         $reportObject = Report::createError(
-            __(
-                '%s line(s) contain(s) an error and cannot be imported',
-                count($report)
-            )
+            __('%s line(s) contain(s) an error and cannot be imported', count($report))
```

Comment:
Not needed change, lets revert.

https://github.com/oat-sa/extension-tao-itemqti/pull/1755/files#r643008991